src/components/article.jsx

```diff
@@ -15,20 +17,42 @@ const Article = () => {
         })
     }, [article_id, setArtLoading]);
 
+    useEffect(() => {
+        fetchComments(article_id).then((data) => {
+        setComments(data);
+        setComLoading(false);
+        })
+    }, [article_id, setComLoading]);
+
     if (artLoading) {
         return <p>Loading...</p>
     }
 
+    if (comLoading) {
+        return <p>Loading...</p>
+    }
+
     return (
         <div>
-            <section>
+            <section className={'singleArticle'}>
                 <h2>{article.title}</h2>
                 <p>Written by {article.author}</p>
                 <img src={article.article_img_url} alt={`${article.topic}`} className={'responsiveImg'}></img>
                 <p>{article.body}</p>
                 <p>Topic: {article.topic}</p>
                 <p><button>+</button> Votes: {article.votes} <button>-</button></p>
             </section>
+            <section>
+            <h2>Comments</h2>
+            <ul className={'comments'}>
+           {comments.map((comment) => {
```

Comment:
If an article has no comments, this would return nothing. Instead, you should display a message to the front-end user to let them know that the article has no comments, just in case they assume the comments haven't loaded correctly